src/components/CheckBox/CheckBox.styles.ts

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
+
+import { TodoType } from '@pages/todos';
+import { theme } from '../../styles/theme.styles';
```

Comment:
```suggestion
import { theme } from '@theme';
```
since `theme` is going to be used frequently, you can declare an alias for it. Although I saw above that you defined your theme using emotion `Theme` so you should be able to read `theme` from the props directly like this:
```typescript
   ${({theme}) => ....}

Reply:
this one worked, is it ok?
`export const StyledTodoInput = styled.input<Theme>`
  width: 100%;
  padding: 10px 20px;
  margin-block: 20px;
  font-size: 20px;
  border: 1px solid ${({ colors }) => colors.grey};
  border-radius: 5px;
`;`

Reply:
is `color.grey` type checked before being passed here?

Reply:
if it is, it's ok